docs/extra-build-config.md

```diff
@@ -74,6 +74,20 @@ To remove (or adjust) this delay set these variables in local.conf:
     BOOT_DELAY = "0"
     BOOT_DELAY_MS = "0"
 
+## Boot media
+
+The Raspberry Pi 4 board can load the boot image files from SD card and USB memory.
+By default SD card media is used as boot media.
+
+To switch the boot media from SD card the following variables are supported
+in local.conf: `CMDLINE_ROOT_PARTITION` and `BOOT_MEDIA`.
```

Comment:
What are possible allowed values for BOOT_MEDIA should be documented as well.  Also describe how boot media and root partition names are connected.

Reply:
That has been dealt with below. Thanks @harunobukurokawa .